workspace/locales/en.yml

```diff
@@ -55,12 +55,22 @@ en:
           list: "Here are the artifacts of %{build}:\n%{artifacts}"
           empty: "No artifact was found in the given build number (%{buildnumber})!"
       jenkins:
+        error:
+          request: Error fetching Jenkins build types
         help:
           build_jenkins:
             syntax: "Jenkins: `build_jenkins` `job_name` [`params`]"
             desc: "Build now a job on Jenkin"
+          jenkins_queue:
+            syntax: "*@buildbot jenkins_queue*"
+            desc: "Lists all queued builds on Jenkins"
+            syntax_wild: "*@buildbot jenkins_queue `some_text*`*"
+            desc_wild: "Lists queued builds Jenkins matching a case-sensitive wildcard search with `*`"
         build_jenkins:
           building: "The %{job_name} is building, please wait...."
           response: "The job is builded in: <%{build_url}| Build #%{queue_result}>"
           failed: "The job failed to build now"
           invalid_job: "The job does not exist!"
+        runningbuildtypes:
```

Comment:
should upercase the first letter in second character or use "_" between characters